### PR TITLE
feat: added a web page generator

### PR DESCRIPTION
**Pull request by fredevb**
Added functionality to generate a web page that can display, select, and combine SVG files. Furthermore, the SVG rendering order can be customized during generation.

image

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -48,6 +48,11 @@ if ( ACTSVG_BUILD_META )
    add_subdirectory(meta)
 endif()
 
+option(ACTSVG_BUILD_WEB "Build the webpage builder interface of ACTSVG" ON )
+if ( ACTSVG_BUILD_META )
```
Comment (asalzburger) [inline CMakeLists.txt:52]: shouldn't that option be `ACTSVG_BUILD_WEB` ?

File: web/include/actsvg/web/tools/scripts/generate_header.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+js_path = os.path.join(current_directory, "template", "script.js")
+html_path = os.path.join(current_directory, "template", "index.html")
+css_path = os.path.join(current_directory, "template", "styles.css")
+
+namespace = "actsvg::web"
+file_text_variables = [("index_text", html_path), ("script_text", js_path), ("css_text", css_path)]
+
+license = r"""// This file is part of the actsvg packge.
+//
+// Copyright (C) 2022 CERN for the benefit of the ACTS project
```
Comment (asalzburger) [inline web/include/actsvg/web/tools/scripts/generate_header.py:23]: 2023, or - if auto - created you can insert the year ?

**Review by asalzburger — CHANGES_REQUESTED** [review]
2 minor updates.

**Comment by fredevb** [comment]
I have applied the changes and made the generated web page directory include rebuild.py :)

**Pull request merged**